chapters/qa-1.mdx

```diff
@@ -74,21 +74,21 @@ Možná struktura je tato:
+Náš cvičný blog není jen o tom ukázat ti, jak to může vypadat. Když se podíváš do kódu, uvidíš v něm nové značky, které jsme popsali [v taháku](https://docs.google.com/spreadsheets/d/e/2PACX-1vTDlg49baGZN9OjApjGlXb4ND9biVc2E2ppfk20iSSPUn6hVlQh1wXSTpG6Dyq_FE-UpmcsmSvdSRmb/pubhtml) spolu s CSS vlastnostmi, které jsme použili. V blogu se nachází flexbox, což je soubor css vlastností, které ti umožní určit umístění obsahu v jednom směru (vertikálně / horizontálně). Pro bližší porozumění si zkus umístit všechny žabičky na lekníny ve **[Flexbox froggy](https://flexboxfroggy.com/#cs)**. Hezká ukázka je i na [css-tricks](https://css-tricks.com/snippets/css/a-guide-to-flexbox/), ale tento materiál je pouze v angličtině.
 
 ---
 
 Poznámka k obrázkům:
 
-Obrázky mohou být obrovské. To zpomaluje stránky a odrazuje jejich uživatele. Ideální je použít vhodný formát obrázku - můžeš si s tím pohrát na [Squoosh](https://squoosh.app/). Nejlepší je nyní používat Webp či Avif, které jsou velmi efektivní. Je třeba si však uvědomit, kdo jsou tví uživatelé. Bylo by mrzuté, kdyby neviděli nic z tvé galerie. Podporu pro tyto obrázky nalezneš na [Can I use](https://caniuse.com/) - když zadáš např. [Webp](https://caniuse.com/?search=webp), zjistíš v jakých prohlížečích jsou podporované. Standardem je podporovat poslední dvě verze prohlížečů - ulehčí ti to práci a prohlížeče se nyní samy aktualizují.
+Obrázky mohou být obrovské. To zpomaluje stránky a odrazuje jejich uživatele. Ideální je použít vhodný formát obrázku - můžeš si s tím pohrát na [Squoosh](https://squoosh.app/). Nejlepší je nyní používat Webp či Avif, které jsou velmi efektivní. Je třeba si však uvědomit, kdo jsou tví uživatelé. Bylo by mrzuté, kdyby neviděli nic z tvé galerie. Podporu pro tyto obrázky nalezneš na [Can I use](https://caniuse.com/) - když zadáš např. [Webp](https://caniuse.com/?search=webp), zjistíš, v jakých prohlížečích jsou podporované. Standardem je podporovat poslední dvě verze prohlížečů - ulehčí ti to práci a prohlížeče se nyní samy aktualizují.
 
 ---
 
-Neboj se si svým blogem pohrát. Nabereš tak zkušenosti, které zúročíš v pozdějších částech kurzu. V kapitole osm zjistíš, jak složku nahrát na Github (jak to máme my), kde se na to může podívat třeba i tvůj možný budoucí zaměstnavatel. Prozatím si ji můžeš uložit v nějakém cloudu jako je Google Drive, One Drive nebo Dropbox. Pokud si nebudeš vědět rady, klidně se na nás obrať na našem [Discordu](https://discord.gg/QbYswwYUPU).
+Neboj se si svým blogem pohrát. Nabereš tak zkušenosti, které zúročíš v pozdějších částech kurzu. V kapitole osm zjistíš, jak složku nahrát na Github (jak to máme my), kde se na to může podívat třeba i tvůj možný budoucí zaměstnavatel. Prozatím si ji můžeš uložit v nějakém cloudu, jako je Google Drive, One Drive nebo Dropbox. Pokud si nebudeš vědět rady, klidně se na nás obrať na našem [Discordu](https://discord.gg/QbYswwYUPU).
```

Comment:
Can we add a link to the Github repository?

Reply:
Done.